_data/simple-icons.json

```
@@ -17044,6 +17044,12 @@
             "hex": "DA074A",
             "source": "https://commons.wikimedia.org/wiki/File:VOX_Logo_2013.svg"
         },
+        {
+            "title": "VRChat",
+            "hex": "000000",
+            "source": "https://hello.vrchat.com",
```

Hmm I'm not seeing SVGs on https://hello.vrchat.com/. The link you have for `guidelines` has a link to the presskit as well, did you mean that?
```suggestion
            "source": "https://hello.vrchat.com/press",
```

---

Icon is a perfect match for the logo file I found at that press link, I'll update it.